R/module_nested_tabs.R

```diff
@@ -243,7 +243,7 @@ srv_nested_tabs.teal_module <- function(id, datasets, modules, is_module_specifi
     }
 
     if (is_arg_used(modules$server, "data")) {
-      data <- .datasets_to_data(modules, datasets, trigger_data)
+      data <- reactive(.datasets_to_data(modules, datasets, trigger_data))
```

Comment:
`trigger_data` isn't used in `.datasets_to_data`
```suggestion
      data <- eventReactive(trigger_data(), .datasets_to_data(modules, datasets))
```

Reply:
What about 
```
  data <- sapply(
    datanames,
    function(x) eventReactive(trigger_data(), datasets$get_data(x, filtered = TRUE)),
    simplify = FALSE
  )
```